src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Customizations/Skillsets/Models/NamedEntityRecognitionSkillLanguage.cs

```diff
@@ -6,28 +6,14 @@ namespace Microsoft.Azure.Search.Models
 {
     using Newtonsoft.Json;
     using Serialization;
+    using System;
 
     /// <summary>
     /// Defines the format of NamedEntityRecognitionSkill supported language codes.
     /// </summary>
     [JsonConverter(typeof(ExtensibleEnumConverter<NamedEntityRecognitionSkillLanguage>))]
     public sealed class NamedEntityRecognitionSkillLanguage : ExtensibleEnum<NamedEntityRecognitionSkillLanguage>
     {
-        /// <summary>
```

Comment:
In fact, the entire type should be marked as `Obsolete`.

Reply:
Will do (the entire type is already marked as Obsolete)
My only reasoning for removal was because those languages are not supported, but then again, if this entire type is Obsolete, this doesn't matter...

Reply:
Also, removing fields is a breaking change. The idea is to make them `[Obsolete]` for one release cycle before removing them outright.